app/pubspec.lock

```diff
@@ -85,20 +71,20 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "2.1.1"
-  cross_file:
+  crypto:
```

Comment:
You changes do not make sense. You've removed cross-file and replaced it with crypto. Do we even need this?
Additionally, you use csslib over crypto later. 

- [ ] Remove all of these if we don't need them

Reply:
flutter pub upgrade made these changes